ManagementPacks/Community.DataOnDemand/Scripts/Get-Netstat.ps1

```diff
@@ -165,7 +165,12 @@ foreach ($line in $results) {
         $dnsName = $dnsCache[$addrs[2]]
     }
 
-    # CSV escape procDesc
+    # CSV escape procDesc and shorten
+    $maxlength = 128
+    $procDesc = $procDesc.Substring(0, [System.Math]::Min($maxlength,$procDesc.length))
+    if ($procDesc.length -eq $maxlength -and $procDesc -notmatch '\.{3}$') {
+        $procDesc = "$procDesc..."
+    }
```

Comment:
Agreed, the logic evolved over a some iterations and didn't take a step back and see it could be simplified.  Needed to add the regex though as assuming it's always safe to add "..." could lead to some ugly results.